lib/services/license.go

```diff
@@ -84,22 +84,27 @@ func UnmarshalLicense(bytes []byte) (License, error) {
 }
 
 // MarshalLicense marshals the License resource to JSON.
-func MarshalLicense(license License, opts ...MarshalOption) ([]byte, error) {
+func MarshalLicense(l License, opts ...MarshalOption) ([]byte, error) {
```

Comment:
nit: you can use `license` as the variable name to keep this readable. See below.